awx/settings/defaults.py

```diff
@@ -940,6 +943,15 @@ def IS_TESTING(argv=None):
             'backupCount': 5,
             'formatter':'simple',
         },
+        'job_lifecycle': {
+            'level': 'INFO',
+            'class':'logging.handlers.RotatingFileHandler',
```

Comment:
You might want to sync up w/ @chrismeyersfsu on his logrotate work, as he's been planning on removing these rotating file handlers.

Reply:
This definitely needs an update now that @chrismeyersfsu has merged his log rotation PR, but I don't expect anything super complicated.